There are some warnings ` warning NU1902: Package 'OpenTelemetry.Api' 1.15.0 has a known moderate severity vulnerability` after creating blazor-wasm-servicedefaults project

**REGRESSION INFO:**  New feature on 11.0 P4

**INSTALL STEPS:**
1.  Clean Win 11 25h2 x64
1.  Install Latest 11.0 P4 build from https://release.dot.net/
1. Apply Feed 

**Platform**
- [x] Windows
- [ ] macOS
- [ ] Linux

**Repro Steps**
1. Open up a normal Command Prompt (Terminal)
1. Create a blazor-wasm-servicedefaults app and run it.
    ```
    dotnet new blazor-wasm-servicedefaults  -o servicedefaults
    ```

**Actual Result**
<!--- Describe the failure that occurred and at one point in the reproduction steps it happened. --->
There are some warnings ` warning NU1902: Package 'OpenTelemetry.Api' 1.15.0 has a known moderate severity vulnerability`
   <img width="1060" height="412" alt="Image" src="https://github.com/user-attachments/assets/a87b028d-90c5-45c0-ab37-3801cb574474" />

   <img width="1483" height="356" alt="Image" src="https://github.com/user-attachments/assets/f4d44332-8fc7-4db5-854b-436787a0d272" />

**Error Log**
```
Restore succeeded with 4 warning(s) in 1.9s
    C:\Users\v-zhajin\blazor-wasm-servicedefaults\blazor-wasm-servicedefaults.csproj : warning NU1603: blazor-wasm-servicedefaults depends on Microsoft.Extensions.ServiceDiscovery (>= 10.4.1) but Microsoft.Extensions.ServiceDiscovery 10.4.1 was not found. Microsoft.Extensions.ServiceDiscovery 10.5.0 was resolved instead.
    C:\Users\v-zhajin\blazor-wasm-servicedefaults\blazor-wasm-servicedefaults.csproj : warning NU1902: Package 'OpenTelemetry.Api' 1.15.0 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-g94r-2vxg-569j
    C:\Users\v-zhajin\blazor-wasm-servicedefaults\blazor-wasm-servicedefaults.csproj : warning NU1902: Package 'OpenTelemetry.Exporter.OpenTelemetryProtocol' 1.15.0 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-mr8r-92fq-pj8p
    C:\Users\v-zhajin\blazor-wasm-servicedefaults\blazor-wasm-servicedefaults.csproj : warning NU1902: Package 'OpenTelemetry.Exporter.OpenTelemetryProtocol' 1.15.0 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-q834-8qmm-v933
Restore succeeded.

```

**Expected Result**
There is no any warning.


**Note:**
1. After updating the packages version, it will work. 

   <img width="1340" height="533" alt="Image" src="https://github.com/user-attachments/assets/bb86fd9d-f0ef-4e50-946c-f985b88bcc28" />

--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -164,10 +164,10 @@
     <NewtonsoftJsonVersion>13.0.4</NewtonsoftJsonVersion>
     <NSwagApiDescriptionClientVersion>13.0.4</NSwagApiDescriptionClientVersion>
     <!-- OpenTelemetry packages for Blazor WASM Service Defaults -->
-    <OpenTelemetryExporterOpenTelemetryProtocolVersion>1.15.0</OpenTelemetryExporterOpenTelemetryProtocolVersion>
-    <OpenTelemetryExtensionsHostingVersion>1.15.0</OpenTelemetryExtensionsHostingVersion>
-    <OpenTelemetryInstrumentationHttpVersion>1.15.0</OpenTelemetryInstrumentationHttpVersion>
-    <OpenTelemetryInstrumentationRuntimeVersion>1.15.0</OpenTelemetryInstrumentationRuntimeVersion>
+    <OpenTelemetryExporterOpenTelemetryProtocolVersion>1.15.3</OpenTelemetryExporterOpenTelemetryProtocolVersion>
+    <OpenTelemetryExtensionsHostingVersion>1.15.3</OpenTelemetryExtensionsHostingVersion>
+    <OpenTelemetryInstrumentationHttpVersion>1.15.3</OpenTelemetryInstrumentationHttpVersion>
+    <OpenTelemetryInstrumentationRuntimeVersion>1.15.3</OpenTelemetryInstrumentationRuntimeVersion>
     <PhotinoNETVersion>2.5.2</PhotinoNETVersion>
     <MicrosoftPlaywrightVersion>1.58.0</MicrosoftPlaywrightVersion>
     <PollyExtensionsHttpVersion>3.0.0</PollyExtensionsHttpVersion>

